app/controllers/api/physical_servers_controller.rb

```diff
@@ -0,0 +1,53 @@
+module Api
+  class PhysicalServersController < BaseController
+    def blink_loc_led_resource(type, id, _data)
+      change_resource_state(:blink_loc_led, type, id)
+    end
+
+    def turn_on_loc_led_resource(type, id, _data)
+      change_resource_state(:turn_on_loc_led, type, id)
+    end
+
+    def turn_off_loc_led_resource(type, id, _data)
+      change_resource_state(:turn_off_loc_led, type, id)
+    end
+
+    #
+    # Name: power_on_resource
```

Comment:
I'm not sure these comments are adding anything - could we remove or otherwise make more verbose to justify why it's needed?